### ISSUE TITLE
When selecting auto start the video path has `true` inserted before the actual path 

### ISSUE THREAD

**Issue by RyanNerd** [ghostlexly/gpu-video-wallpaper#8]
For example this is what the startup command looks like:

`bash -c '"/home/ryan/.local/bin/video-wallpaper.sh" --start  "true /home/ryan/Videos/2160p.mp4"'`

Which was throwing this error:
```
The legacy option syntax ('-wid value') is deprecated and dangerous.
Please use '--wid=value'.
[file] Cannot open file 'true /home/ryan/Videos/2160p.mp4': No such file or directory
Failed to open true /home/ryan/Videos/2160p.mp4.


Exiting... (Errors when loading file)
mpv died, exit status 2
```

The work-around is of course to remove the `true ` from the video file path

**Comment by ZediAlreadyTaken** [comment]
I am trying to work my brain around this stuff, but it seems like the $LAUNCH_SCRIPT variable picks up the wrong path because of bad configuration. The work-around would be to fix `${@:2}` in line 64 of video-wallpaper.sh... Please have a look..

Edit: I fixed it, please refer to line 64 of file `video-wallpaper.sh` in my fork

Edit2: Last night, I issued a pull request about this, forgot to update this comment.

**Comment by SwallowYourDreams** [comment]
I've addressed this issue a while ago in my fork. Just forgot to issue a PR for this repo. Did so a couple of minutes ago.

**Comment by ghostlexly** [comment]
This issue seems resolved.
Can you confirm @RyanNerd

**Comment by RyanNerd** [comment]
Closed by #14 
Thank you!
